File: CharacterManagerApiTutorial/Models/LookupDataDto.cs
namespace CharacterManagerApiTutorial.Models
{
    public class LookupDataDto
    {
        public List<RaceType> RaceTypes { get; set; } = [];
        public List<ClassType> ClassTypes { get; set; } = [];
        public List<FactionType> FactionTypes { get; set; } = [];
        public List<Realm> Realms { get; set; } = [];
    }
}

File: character-manager-frontend/src/app/models/lookup-data-dto.ts
import { ClassType } from './class-type';
import { RaceType } from './race-type';
import { FactionType } from './faction-type';
import { Realm } from './realm';

export interface LookupDataDto {
  classTypes: ClassType[];
  raceTypes: RaceType[];
  factionTypes: FactionType[];
  realms: Realm[];
}
